From: Hilko Bengen <bengen@debian.org>
Date: Wed, 19 Dec 2012 22:24:28 +0100
Subject: patch: easy fix for compiling with -Werror=format-security

error: format not a string literal and no format arguments [-Werror=format-security]
---
 src/ldm.c |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/ldm.c b/src/ldm.c
index a55723c..a4eb799 100644
--- a/src/ldm.c
+++ b/src/ldm.c
@@ -2759,7 +2759,7 @@ _dm_create_mirrored(const LDMVolumePrivate * const vol, GError ** const err)
         GString * chunk = _dm_create_part(part, cookie, err);
         if (chunk == NULL) {
             if (err && (*err)->code == LDM_ERROR_MISSING_DISK) {
-                g_warning((*err)->message);
+                g_warning("%s", (*err)->message);
                 g_error_free(*err); *err = NULL;
                 g_string_append(target.params, " - -");
                 continue;
@@ -2804,7 +2804,7 @@ out:
         for (int i = devices->len; i > 0; i--) {
             GString *device = g_array_index(devices, GString *, i - 1);
             if (!_dm_remove(device->str, 0, &cleanup_err)) {
-                g_warning(cleanup_err->message);
+                g_warning("%s", cleanup_err->message);
                 g_error_free(cleanup_err); cleanup_err = NULL;
             }
         }
@@ -2847,7 +2847,7 @@ _dm_create_raid5(const LDMVolumePrivate * const vol, GError ** const err)
         GString * chunk = _dm_create_part(part, cookie, err);
         if (chunk == NULL) {
             if (err && (*err)->code == LDM_ERROR_MISSING_DISK) {
-                g_warning((*err)->message);
+                g_warning("%s", (*err)->message);
                 g_error_free(*err); *err = NULL;
                 g_string_append(target.params, " - -");
                 continue;
@@ -2892,7 +2892,7 @@ out:
         for (int i = devices->len; i > 0; i--) {
             GString *device = g_array_index(devices, GString *, i - 1);
             if (!_dm_remove(device->str, 0, &cleanup_err)) {
-                g_warning(cleanup_err->message);
+                g_warning("%s", cleanup_err->message);
                 g_error_free(cleanup_err); cleanup_err = NULL;
             }
         }
